src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Popup/PopupRenderer.uwp.cs

```diff
@@ -0,0 +1,275 @@
+
+			flyoutStyle.Setters.Add(new Windows.UI.Xaml.Setter(FlyoutPresenter.BackgroundProperty, Element.Color.ToWindowsColor()));
+
+#if UWP_18362
+			if (Element.Color == Color.Transparent)
+				flyoutStyle.Setters.Add(new Windows.UI.Xaml.Setter(FlyoutPresenter.IsDefaultShadowEnabledProperty, false));
+#endif
+		}
+
+		void ApplyStyles()
+		{
+			Control.Style = panelStyle;
+			FlyoutPresenterStyle = flyoutStyle;
+		}
+
+		void Show()
+		{
+			if (Element.Anchor != null)
+			{
+				var anchor = Platform.GetRenderer(Element.Anchor).ContainerElement;
+				FlyoutBase.SetAttachedFlyout(anchor, this);
+				FlyoutBase.ShowAttachedFlyout(anchor);
+			}
+			else
+			{
+				var frameworkElement = Platform.GetRenderer(Element.Parent as VisualElement).ContainerElement;
+				FlyoutBase.SetAttachedFlyout(frameworkElement, this);
+				FlyoutBase.ShowAttachedFlyout(frameworkElement);
+			}
+
+			Element.OnOpened();
+		}
+
+		void SetDialogPosition(LayoutOptions verticalOptions, LayoutOptions horizontalOptions)
+		{
+			if (IsTopLeft())
+				Placement = FlyoutPlacementMode.TopEdgeAlignedLeft;
+			else if (IsTop())
+				Placement = FlyoutPlacementMode.Top;
+			else if (IsTopRight())
+				Placement = FlyoutPlacementMode.TopEdgeAlignedRight;
+			else if (IsRight())
+				Placement = FlyoutPlacementMode.Right;
+			else if (IsBottomRight())
+				Placement = FlyoutPlacementMode.BottomEdgeAlignedRight;
+			else if (IsBottom())
+				Placement = FlyoutPlacementMode.Bottom;
+			else if (IsBottomLeft())
+				Placement = FlyoutPlacementMode.BottomEdgeAlignedLeft;
+			else if (IsLeft())
+				Placement = FlyoutPlacementMode.Left;
+			else if (Element.Anchor == null)
+				Placement = FlyoutPlacementMode.Full;
+			else
+				Placement = FlyoutPlacementMode.Top;
+
+			bool IsTopLeft() => verticalOptions.Alignment == LayoutAlignment.Start && horizontalOptions.Alignment == LayoutAlignment.Start;
+			bool IsTop() => verticalOptions.Alignment == LayoutAlignment.Start && horizontalOptions.Alignment == LayoutAlignment.Center;
+			bool IsTopRight() => verticalOptions.Alignment == LayoutAlignment.Start && horizontalOptions.Alignment == LayoutAlignment.End;
+			bool IsRight() => verticalOptions.Alignment == LayoutAlignment.Center && horizontalOptions.Alignment == LayoutAlignment.End;
+			bool IsBottomRight() => verticalOptions.Alignment == LayoutAlignment.End && horizontalOptions.Alignment == LayoutAlignment.End;
+			bool IsBottom() => verticalOptions.Alignment == LayoutAlignment.End && horizontalOptions.Alignment == LayoutAlignment.Center;
+			bool IsBottomLeft() => verticalOptions.Alignment == LayoutAlignment.End && horizontalOptions.Alignment == LayoutAlignment.Start;
+			bool IsLeft() => verticalOptions.Alignment == LayoutAlignment.Center && horizontalOptions.Alignment == LayoutAlignment.Start;
+		}
+
+		SizeRequest IVisualElementRenderer.GetDesiredSize(double widthConstraint, double heightConstraint)
+		{
+			if (isDisposed || Control == null)
+				return new SizeRequest();
```

Comment:
Roslyn suggestion.
```suggestion
				return default(SizeRequest);
```

Reply:
Easy enough to fix, I'll update this in the next code push